{{cookiecutter.project_dirname}}/terraform/main.tf

```diff
@@ -89,9 +89,10 @@ resource "kubernetes_secret_v1" "env" {
 
   data = { for k, v in merge(
     {
-      DJANGO_SECRET_KEY = random_password.django_secret_key.result
-      EMAIL_URL         = var.email_url
-      SENTRY_DSN        = var.sentry_dsn
+      DJANGO_SECRET_KEY        = random_password.django_secret_key.result
+      DJANGO_ENABLE_API_DOC    = var.environment_slug == "prod" ? "False" : "True"
```

Comment:
I leave a generic comment in this point as "food for thought" on what is the best way to integrate this functionality into the template (Talos), which I think is a great starting point.
But I'm not so sure that integrating it by default in the template could be the best solution, perhaps the most suitable solution could be a public package on a separate repository that can be integrated into the Talos backend in Django with a simple `pip install` and adding it to the `INSTALLED_APPS` (?)

Reply:
The reason we're embedding the API doc by default is that, this way, the API can be inspected and tested by our QA testers, as well.